### PR TITLE
Feat/evm backend

### PR DESCRIPTION
**Pull request by leisterfrancisco**
### What does this PR do?

- add hapi evm logic to logic related to Antelope EVM implementations

### CODE REVIEW

**Comment by leisterfrancisco** [comment]
Please ignore comments of CodeFactor, I'll fix them in the following PR which will include more logic related to sync with old tEVM blocks

File: hapi-evm/package.json [hunk 1]
```diff
@@ -0,0 +1,49 @@
+{
+  "name": "backend-boilerplate",
```
Comment (xavier506) [inline hapi-evm/package.json:2]: "name": "evm-backend",

File: hapi-evm/package.json [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+  "version": "1.0.0",
+  "description": "AntelopeIO Boilerplate for backend development",
```
Comment (xavier506) [inline hapi-evm/package.json:4]: EVM Statistics Backend Service

**Review by xavier506 — COMMENTED** [review]
@leisterfrancisco please see codefactor comments

**Review by Torresmorah — APPROVED** [review]
LGTM

**Pull request merged**